serializer.py

```diff
@@ -0,0 +1,72 @@
+# UNTESTED WITH API
+import json
+
+
+class Link(object):
+    def __init__(self, resource_type, resource_name):
+        self.s = {'type': resource_type,
+                  'id': resource_name}
+
+
+class Resource(object):
+    def __init__(self, resource_type, attributes, relationships):
+        self.s = {}
+        self.s['data'] = {}
```

Comment:
Technically, the object with the `data` key is the document; it's only the value at `data` that's a resource. 

But the document object also has other fields besides `data` (e.g. `meta`), though we're not using any of them right now. So, it might make sense to add a `Document` class so that, if we want to use the other fields in the future, we don't have to rewrite all the code that's using this serializer.